### PR TITLE
Ddf ee api

### PR DESCRIPTION
**Pull request by benwulfe**
This PR makes minor streamlined fixes to the earth engine APIs that we have created, unifying them to be accessed under a single python module and introducing a colab that will be linked from the documentation page describing how to use the API.

### CODE REVIEW

File: dataset.py
Comment (kazob1998) [inline dataset.py:1]: Nice catch!

**Review by kazob1998 — APPROVED** [review]

**Comment by kazob1998** [comment]
LGTM!

File: ddf_ee_api.ipynb
Comment (rothn) [inline ddf_ee_api.ipynb:1]: Did you mean to include an image here?

Comment (rothn) [inline ddf_ee_api.ipynb:1]: nit: s/"is can be imported"/"can be imported"

Comment (rothn) [inline ddf_ee_api.ipynb:1]: nit: s/gmail address/Google account

File: eeddf.py [hunk 1]
```diff
@@ -1,21 +1,60 @@
...
     _initialized = True
 
     import google
     from google.colab import auth
     auth.authenticate_user()
 
     credentials, project_id = google.auth.default()
-    ee.Initialize(credentials, project='river-sky-386919', opt_url='https://earthengine-highvolume.googleapis.com')
+    if _test_environment:
+      ee.Initialize(credentials, project='river-sky-386919', opt_url='https://earthengine-highvolume.googleapis.com')
```
Comment (rothn) [inline eeddf.py:51]: Another more general way to parameterize this might be to store the project ID in a global and as an argument.

**Review by rothn — APPROVED** [review]
A few minor questions and suggestions. Nothing that should block a merge.

File: ddf_ee_api.ipynb
Comment (benwulfe) [reply ddf_ee_api.ipynb:1]: should be an image...

File: eeddf.py (hunk 1 above)
Comment (benwulfe) [reply eeddf.py:51]: thats a good idea... going to change to support this...

File: ddf_ee_api.ipynb
Comment (rothn) [reply ddf_ee_api.ipynb:1]: LGTM-- verified in reviewnb.

**Pull request merged**